Capability map supporting pillar (#117)
sa-tre/satre-specification: docs/source/pillars/supporting.md

:alt: SATRE Pillars Capability Map
:align: center

SATRE Pillars Capability Map
```

<!-- See all pillars of the SATRE Pillars Capability Map here: {ref}`satre_pillars` -->

## Business continuity management

_What the organisation does to ensure the development, testing, and maintenance of business continuity plans._

```{list-table}
:header-rows: 1
:name: tab-business-continuity-subject

* - Statement
  - Guidance
  - Importance
* - You have a business continuity plan that includes consideration of loss of service for deployed TREs
  - This may be due to downtime from service providers, a breach, or loss of power. Your plan should detail your process for managing loss of service for deployed TREs, and evaluation of impact of such loss
  - Recommended
* - You regularly test the aspects of your business continuity plan concerning TREs, and have a process in place to iterate the plan if required
  -
  - Recommended
```

## Project and programme management

_What the organisation does to ensure effective management of programmes and projects._

```{list-table}
:header-rows: 1
:name: tab-project-programme-management

* - Statement
  - Guidance
  - Importance

> * - All projects requiring TRE are assigned project management support

Reviewer: "project management" can mean a lot of things (as can "project"). Is there another term we can use, since most research projects will already have their own project manager?

> * - All projects requiring TRE are assigned project management support
>   - This may include budget management, tracking TRE status, managing communications with the TRE operations team, and other project support tasks
>   - Recommended

Reviewer 2: Closer but maybe not quite right! Maybe:
```suggestion
* - All projects requiring a TRE have a named project manager
  - The project manager has responsibility to ensure the smooth running of the project. Their responsibilities may include budget management, tracking TRE status, managing communications with the TRE operations team, and other project support tasks
  - Recommended
```

> * - All projects requiring TRE are assigned project management support
>   - This may include budget management, tracking TRE status, managing communications with the TRE operations team, and other project support tasks
>   - Recommended

Reviewer 2: Cool, this is useful steer
```suggestion
* - The TRE operators should ensure that all projects requiring a TRE have a named project manager
  - The project manager has responsibility to ensure the smooth running of the project. Their responsibilities may include budget management, tracking TRE status, managing communications with the TRE operations team, and other project support tasks
  - Recommended
```

> * - Project management support assigned to a project requiring a TRE should not have direct access to the TRE

Reviewer 3: ```suggestion
* - Project managers should not have direct access to the TRE
```

Reviewer 2: Happy to commit, as long as we define project manager somewhere (glossary?)

  - Doing so ensures a separation between those able to access sensitive data, and those overseeing access to sensitive data
  - Optional
```

## Knowledge management

_What the organisation does to acquire, enrich, share, store, publish and enhance the expertise across the whole organisation._

```{list-table}
:header-rows: 1
:name: tab-knowledge-management

* - Statement
  - Guidance
  - Importance

> * - All features of your TRE infrastructure requiring input or attention from any stakeholders must be well documented

Reviewer: ```suggestion
* - All features of your TRE infrastructure must be well documented
```
Are there any features that don't need to be well documented?

  - This includes ensuring all documentation is discoverable, clear, and able to be easily updated based on stakeholder feedback
  - Mandatory
* - You should have an education programme in place to upskill stakeholders in the use and management of your TRE infrastructure
  - This may include learning modules, workshops and other resources on how to effectively access and use a TRE, FAQs pages, and accessible pathways for additional support
  - Recommended
* - You should periodically carry out a training needs analysis (TNA) for all stakeholders included within your TRE provision
  - At least once every 12 months you should assess the training needs of your stakeholders, and ensure they have easy access to all required training materials
  - Recommended
```

## Financial management

_All activities aimed at the efficient and effective management of money (funds) in such a manner as to allow the organisation to accomplish its objectives._

```{list-table}
:header-rows: 1
:name: tab-financial-management

* - Statement
  - Guidance
  - Importance

> * - All projects requiring a TRE must know and accept the financial cost of doing so

Reviewer: Isn't this a requirement/capability of the user/researcher, rather than the TRE? Should this actually be `All TREs must know and accept the financial cost of doing so`?

Reviewer 2: > Isn't this a requirement/capability of the user/researcher, rather than the TRE?

Are you saying it should be a requirement of the user/researcher, rather than TRE? If so I read what it already says as that, and your rewording as the opposite (it would then be a requirement of TRE teams

Reviewer 3: @manics see my suggestion below (not sure why it didn't show up as a reply here)

> * - All projects requiring a TRE must know and accept the financial cost of doing so
>   - Costs may include provision of the underlying TRE infrastructure, additional resources required in a specific TRE (for instance memory or additional compute), hardware including managed devices, and staff support costs
>   - Mandatory

Reviewer 3: How about:

```suggestion
* - All projects using a TRE must have a plan for paying for any financial costs associated with their TRE use
  - Costs may include provision of the underlying TRE infrastructure, additional resources required in a specific TRE (for instance memory or additional compute), hardware including managed devices, and staff support costs
  - Mandatory
```

Reviewer 2: Ah I see the diff now - are we trying to make this as centred on TRE teams as poss (i.e. not hold researchers/users liable for requirements).

If so happy to commit, just need to ensure we define TRE operators somewhere

Reviewer 2: (Added as a term to add to glossary, along with project manager & project management

> * - You should be able to track costs associated with each TRE project
>   - This includes knowing which costs are associated with which project, and having an appropriate charging mechanism in place in line with your organisational policy
>   - Recommended

Reviewer 3: ```suggestion
* - The organisation operating the TRE should be able to track costs associated with each TRE project
  - This includes knowing which costs are associated with which project, and having an appropriate charging mechanism in place in line with your organisational policy
  - Recommended
```

* - You should have a process in place to ensure your TRE provision remains sustainable
  - This could include having a cost recovery process in place, or setting up a long-term funding mechanism to support projects with TREs. At any given time, you should have funds free to cover all potential foreseen TRE provision for at least 12 months
  - Recommended
* - You should minimise the cost of your TRE infrastructure wherever possible
  - You should have regular reviews of your TRE provision and actively work to bring down costs, streamline provision, and optimise support
  - Optional
```

## Procurement

_What the organisation does to ensure the effective sourcing, purchasing and supply of the goods and services that enable the organisation to operate._

```{list-table}
:header-rows: 1
:name: tab-procurement

* - Statement
  - Guidance
  - Importance

> * -
>   -
>   -

Reviewer 3: ```suggestion
* - The organisation operating the TRE must identify any goods or services that will be needed to operate the TRE and ensure that a plan is in place to purchase them as needed.
  - These may include computing hardware, cloud credits or devices through which users access the TRE.
  - Mandatory
```

```

## IT Service management

_The implementation and management of quality IT services that meet the needs of the organisation._

```{list-table}
:header-rows: 1
:name: tab-it-service-management

* - Statement
  - Guidance
  - Importance

> * - You must have a team in place to support projects working with TREs

Reviewer: Is this separate from the "project management" support listed earlier?

Reviewer 2: I see it as yes, one is ensuring the project is running smoothly, the other is to ensure there is technical support in place

  - This may be part of your organisation's IT support team, or separate. Responsibility should be clear and stakeholders should easily be able to access support appropriate to their needs
  - Mandatory
```

## Relationship management

_All activities aimed at ensuring a continuous level of engagement is maintained between the organisation and its customers, stakeholders & other interested parties._

### Stakeholder relationships

```{list-table}
:header-rows: 1
:name: tab-relationship-stakeholder

* - Statement
  - Guidance
  - Importance
* - You should have a clear process in place for stakeholders to feedback on your TRE infrastructure
  - This may include a GitHub repository where people can open issues and discussions, communication streams like Slack or email, or forms stakeholders can fill in
  - Recommended
```

### Patient and Public Engagement

_What the organisation does to ensure it establishes and maintains mutually beneficial relationships between the organisation and the public in order to maintain trust._

```{list-table}
:header-rows: 1
:name: tab-patient-public-engagement

* - Statement
  - Guidance
  - Importance

> * -
>   -
>   -

Reviewer 3: ```suggestion
* - The TRE organisation should develop a plan for engagement with the public.
  - This may involve consulting members of the public, holding public events or directly including lay members in decision making panels.
  - Recommended
```

Reviewer 2: @jemrobinson seeing @JimMadge 's comment, shall we remove this?

Reviewer 3: Yes

```

## Legal services

_The ability of the TRE organisation to access suitable and timely legal advice._

<!-- Specific requirements? _e.g._ Article 32 of the GDPR requires organisations to regularly test and evaluate the effectiveness of the technical and organisational measures employed to protect personal data, and penetration testing is an effective way of assessing your technical defences. -->

### Legal advisory

_Ability of the organisation to provide suitable and timely legal advice._

```{list-table}
:header-rows: 1
:name: tab-legal-services

* - Statement
  - Guidance
  - Importance

> * -
>   -
>   -

Reviewer 3: ```suggestion
* - The TRE organisation should identify areas where legal advice may be needed and plan where this will be sourced from.
  - It is likely that legal advice will be necessary for several issues around the handling of sensitive data.
  - Recommended
```

Reviewer 2: @jemrobinson already covered here: https://github.com/sa-tre/satre-specification/blob/main/docs/source/pillars/information_governance.md#policy-regulation-and-management?

Reviewer 2: I would agree with that distinction, then think the wording needs to change to smth like: 
```suggestion
* - The TRE organisation should have ready access to legal advice
  - It is likely that legal advice will be necessary for several issues around the handling of sensitive data. TRE organisations should have ready access to legal advice, including a way to solicit advice and carry out associated actions
  - Recommended
```

Reviewer 2: Have added smth about contracts in reference to point below

```

### Data protection

> _Ability to ensure data is used fairly, lawfully and transparently. Used for specified, explicit purposes. used in a way that is adequate, relevant and limited to only what is necessary._

Reviewer: This sounds incomplete.

Reviewer 3: @manics see suggestion below

> _Ability to ensure data is used fairly, lawfully and transparently. Used for specified, explicit purposes. used in a way that is adequate, relevant and limited to only what is necessary._

Reviewer 3: ```suggestion
_Ability to ensure data is used fairly, lawfully and transparently; for specified, explicit purposes; and in a way that is adequate, relevant and limited to only what is necessary._
```

```{list-table}
:header-rows: 1
:name: tab-legal-dp

* - Statement
  - Guidance
  - Importance

> * -
>   -
>   -

Reviewer 3: ```suggestion
* - The TRE organisation should identify areas where data protection advice may be needed and plan where this will be sourced from.
  - It is likely that data protection advice will be necessary for several issues around the handling of sensitive data.
  - Recommended
```

Reviewer 2: Similar, maybe should be updated here: https://github.com/sa-tre/satre-specification/blob/main/docs/source/pillars/information_governance.md#policy-regulation-and-management?

Reviewer 3: Again, policy vs. outside advice.

Reviewer 3: I agree that legal and data protection are v similar. Happy to merge them. Will you make the change? Also needs a change to the diagram I think.

Reviewer 2: Actually I want to back track on that, as e.g .at Turing our new DPM won't be a lawyer, so can be disctinct

```

### Contract management

_What the organisation does to ensure that all contracts are effectively managed within required frameworks._

```{list-table}
:header-rows: 1
:name: tab-legal-cm

* - Statement
  - Guidance
  - Importance

> * -
>   -
>   -

Reviewer 3: ```suggestion
* - The organisation running the TRE should identify who will be responsible for managing contracts related to the TRE.
  - These contracts may include data sharing agreements, secondments of personnel or limitations on how results obtained with the data can be distributed.
  - Recommended
```

Reviewer 2: Should this be helper text here: https://github.com/sa-tre/satre-specification/blob/main/docs/source/pillars/information_governance.md#policy-regulation-and-management

Reviewer 3: Again, I *think* the difference is "do you have a policy for X?" vs. "do you have access to expert advice about X?"
